.vale/styles/Vocab/Che/reject.txt

```diff
@@ -12,8 +13,6 @@ now
 operator
 our
 out of the box
-[Pp]lease
-pod
```

Comment:
`pod` will be reported by Vale as an unknown work AFAIK.

Reply:
This is a change in the `reject.txt` file which was refusing `pod`. The scoping for the vocabulary is not perfect, so for capitalization, the job is moving to the `TechnicalTerms` rule (with a more complete error message).

Reply:
Ack, makes sense